unstructured/partition/auto.py

```diff
@@ -374,6 +374,7 @@ def partition(
             include_page_breaks=include_page_breaks,
             infer_table_structure=infer_table_structure,
             strategy=strategy,
+            generate_extra_info=False,
```

Comment:
`generate_extra_info` could mean anything. What would be a more descriptive name? Is it generate OCR info? generate parsing info? Generate table metadata? The name should better direct the caller as to what to expect.

Reply:
I agree, I changed the name to `generate_inner_elements_info` and added some info in the doc strings. Do you think is better now?

Reply:
Definitely an improvement, that at leasts gives something to look around for (inner-elements) suits me :)